src/components/register/register.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { IRegisterFormProperties } from '../../@types/types';
+import './register.scss';
+
+export function Register(): ReactElement {
+
+  const history = useHistory();
+
+  const [form, setState] = useState<IRegisterFormProperties>({
+    username: '',
+    password: '',
+    email: '',
+  });
+
+  function redirectToLogin() {
+    history.push('/login');
+  }
+
+  const updateField = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setState({
+      ...form,
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  return (
+    <div className="c-register">
+      <h2 className="c-register__header">Register</h2>
+      <form className="c-register__form">
+        <input type="text" placeholder="Username" name="username" value={form.username}
```

Comment:
Another piece of bike shedding, and probably would require @gibula-m's agreement, is that we don't need both of the `Username` and `Email`. They'd both need to be unique in the database in order to satisfy your Login functionality, and are both required? Seems a little pointless to me...

But then again, I'd be keen on not using `Passwords` and self managed users in favor of OpenID such as `Login with Google`, `Login with Facebook`. Another call to make by @gibula-m

Reply:
@paroxp I think we can start with logging in through this data and move to OpenId at a later stage.

Reply:
We can. But that means we have to engineer that solution and at later stage get rid of it :D